src/components/availability-panel/drag-drop-context/DroppableComponent.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC } from 'react';
+import { Droppable, Draggable } from 'react-beautiful-dnd';
+import classNames from '@/components/availability-panel/drag-drop-context/styles.module.scss';
+import { droppableComponent } from '../../constants/types';
+
+const imageGenerator = (name: string) => `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${name}/img.png`;
+
+const getItemStyle = (isDragging: boolean, draggableStyle: any) => {
+  const style = {
+    background: isDragging ? '#d1d1d1' : 'white',
+    ...draggableStyle,
+  };
+  return style;
+};
+
+const DroppableComponent: FC<droppableComponent> = ({
+  droppableId,
+  unAssignedTasks,
+  idleMembers,
+}) => (
+  <div>
+    {droppableId === 'tasks' ? (
+      <Droppable droppableId="tasks" isCombineEnabled>
+        {(provided) => (
+          <div ref={provided.innerRef}>
+            {unAssignedTasks.map((task, index) => (
```

Comment:
NIT: Since we are using only `task.id` and `task.title`, we can destructure the `task` here.